src/inference/src/cache_manager.hpp

```diff
@@ -141,13 +141,16 @@ class FileStorageCacheManager final : public ICacheManager {
                 auto mmap = ov::load_mmap_object(blob_file_name);
                 auto shared_buffer =
                     std::make_shared<ov::SharedBuffer<std::shared_ptr<MappedMemory>>>(mmap->data(), mmap->size(), mmap);
+#if 0
                 OwningSharedStreamBuffer buf(shared_buffer);
                 std::istream stream(&buf);
-                reader(stream);
+#else
+                std::ifstream stream(blob_file_name, std::ios_base::binary);
```

Comment:
We can open simultaneously with mmap and std::ifstream.
It allows to use standard istream instead of OwningSharedStreamBuffer. But it may brings potential performance issues due to concurrent access to one file. My tests didn't show any problems with such approach